eskip/eskip.go

```diff
@@ -769,7 +769,7 @@ func GenerateIfNeeded(existingId string) string {
 	sb.WriteString("route")
 
 	for i := 0; i < randomIdLength; i++ {
-		ai := rand.Intn(len(alphabet))
+		ai := rand.IntN(len(alphabet)) // #nosec
```

Comment:
Why the optout here and why it didn't fail before before? I also think it's better to not update the deprecated function and remove it, if it's too old.

Reply:
There is multiple optouts but I can't figure why it fails now